### PR TITLE
added exchange integration bitcointrade

### PR DESCRIPTION
**Pull request by cryptofortinero**
New Exchange [Bitcointrade](https://www.bitcointrade.com.br/):

We are the first cryptocurrency broker in Brazil to have PCI DSS Compliance certification, an international data security certificate for the payment card industry, awarded to companies that comply with its rules for handling sensitive credit card data, such as such as: number, validity, bearer name and CVV.

We have the same dream of creating the most disruptive, secure and innovative digital currency platform. BitcoinTrade is the safest platform in Brazil to buy and sell Bitcoin and Ethereum

### CODE REVIEW

**Comment by cryptofortinero** [comment]
Hi @kroitor can you help me with travis CI?, I saw the details but the error log said something about exmo exchange

**Comment by kroitor** [comment]
@cryptofortinero that may be intermittent errors with this or that exchange, i've restarted the build.

**Comment by kroitor** [comment]
@cryptofortinero we're missing some of the endpoints: https://apidocs.bitcointrade.com.br/#operation/GetPairs
We must list them all out. The exchange should be usable with just the public API (without keys).

**Comment by cryptofortinero** [comment]
@kroitor okey. I going to add it now

File: js/bitcointrade.js [hunk 1]
```diff
@@ -0,0 +1,687 @@
...
+    }
+
+    parseTrade (trade, market = undefined) {
+        const timestamp = this.parseDate (this.safeString (trade, 'timestamp'));
+        const id = timestamp;
+        let side = this.safeString (trade, 'type');
+        const takerOrMaker = 'taker';
+        if (side !== undefined) {
+            side = side.toLowerCase ();
+        }
```
Comment (kroitor) [inline js/bitcointrade.js:308]: Let's replace:

```JavaScript
        let side = this.safeString (trade, 'type');
        const takerOrMaker = 'taker';
        if (side !== undefined) {
            side = side.toLowerCase ();
        }
```
↓
```JavaScript
        const side = this.safeStringLower (trade, 'type');
        const takerOrMaker = 'taker';
```

**Review by mrbaguvix — APPROVED** [review]

**Comment by ttodua** [comment]
exchange was migrated to ripio

**Pull request closed without merging**